src/main/resources/static/script.js

```diff
@@ -89,11 +85,25 @@ function initMap() {
                 event.preventDefault();
             }
         });
-    }
-
-
 }
 
+    function calculateAndDisplayRoute(directionsService, directionsRenderer) {
```

Comment:
This function is very similar to the calculateAndDisplayRoute function down below. Do we need both? The difference I see is that this one is embedded within initMap(), calls the findPointsOfInterest method, and doesn't include the error message "Sorry, we could not calculate driving directions for these locations. Please try a different location."  I figure we should consolidate these two functions into one, right?